index/scorch/scorch.go

```diff
@@ -401,8 +401,10 @@ func (s *Scorch) Batch(batch *index.Batch) (err error) {
 		analysisResults[itemsDeQueued] = result
 		itemsDeQueued++
 		result.VisitFields(func(f index.Field) {
-			atomic.AddUint64(&s.stats.TotBytesIndexedAfterAnalysis,
-				analysisBytes(f.AnalyzedTokenFrequencies()))
+			if segment.CollectIOStats {
```

Comment:
I would recommend against adding this flag in scorch_segment_api.
Instead make it a scorch config option and apply this only while -
* Estimating usage within bleve
* Reporting usage reported by zap again from bleve

Leave zap's computation on always.